build.gradle

```diff
@@ -126,12 +137,13 @@ jacocoTestReport {
 tasks.test.finalizedBy    tasks.jacocoTestReport
 tasks.sonarqube.dependsOn tasks.jacocoTestReport
 
-def jar = rootProject.name + '-' + version + ".jar"
+ext.bootJarPath    = relativePath(tasks.bootJar.outputs.files.singleFile)
+ext.gitShortCommit = 'git rev-parse --short=8 HEAD'.execute().text.trim()
+ext.ociImageName   = "local/${rootProject.name}"
```

Comment:
Why not local/${parentDirName} to be consistent with lib?

Reply:
Do you mean `basename $(git config --get remote.origin.url) .git` ?
Right now, both values are the same (i.e.  gradle rootProject name = git repository name), but the current implementation can be changed with `'basename $(git config --get remote.origin.url) .git'.execute().text.trim()`.

Reply:
Yes git repo name

Reply:
Patch committed and pushed in every repository, except iexec-common